.github/workflows/kbs-rust.yml

```
@@ -24,8 +24,6 @@ jobs:
       matrix:
         rust:
           - stable
```

The original proposal also included that we fix the version and not use a moving target, i.e.,
```suggestion
          - 1.76.0
```

with these updates, it's also possible to drop the `matrix` construct altogether.